### PR TITLE
Add new product identifier for SDK Lock Example

### PR DESCRIPTION
**Pull request by suryanshup**

### CODE REVIEW

File: src/lib/profiles/vendor/nestlabs/device-description/NestProductIdentifiers.hpp [hunk 1]
```diff
@@ -64,6 +64,7 @@ enum NestWeaveProductId
     kNestWeaveProduct_SDKSampleLightActuator     = 0xFE06,
     kNestWeaveProduct_SDKSampleButtonSensor      = 0xFE07,
     kNestWeaveProduct_SDKSampleMotionSensor      = 0xFE08,
+    kNestWeaveProduct_OneDeviceSDKLockExample    = 0xFE0B,
```
Comment (jaylogue) [inline src/lib/profiles/vendor/nestlabs/device-description/NestProductIdentifiers.hpp]: Can we name this simply "SDKLockExample"?

Comment (suryanshup) [reply src/lib/profiles/vendor/nestlabs/device-description/NestProductIdentifiers.hpp]: Just saw this comment. Yep, will do !

Comment (suryanshup) [reply src/lib/profiles/vendor/nestlabs/device-description/NestProductIdentifiers.hpp]: Done !

**Comment by suryanshup** [comment]
Waiting for Travis Cl build to go green before merging.

**Comment by suryanshup** [comment]
Commit did not have my correct email address causing an issue where googlebot prompted me to sign the CLA. I have reset the commit with the right google email address and the check should not fail anymore.

**Comment by suryanshup** [comment]
UPDATE: Vendor ID for this SDK will be Google.

**Pull request closed without merging**